ip-messaging/users/token-generation-server/token-generation-server.5.x.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
```

Comment:
Are you using `System.Linq`? Is usually a good practice to do not include imports that won't be used in this file. You can use your editor to remove the unused imports.